detection_rules/kbwrap.py

```diff
@@ -358,10 +369,18 @@ def kibana_export_rules(  # noqa: PLR0912, PLR0913, PLR0915
             tactic_name = first_tactic if not no_tactic_filename else None  # type: ignore[reportUnknownMemberType]
             rule_name = rulename_to_filename(rule_resource.get("name"), tactic_name=tactic_name)  # type: ignore[reportUnknownMemberType]
 
+            local_contents = None
             save_path = directory / f"{rule_name}"
+
+            rule = params.get("rule")
+            if rules and rule and rule.get("rule_id") in rules.id_map:
+                save_path = rules.id_map[rule["rule_id"]].path
+                local_contents = rules.id_map[rule["rule_id"]].contents
```

Comment:
nitpick: would you mind making a local var with `rule["rule_id"]`, so that the following lines are easier to glance over?

something like
```python
rule_id = rule["rule_id"]
save_path = rules.id_map[rule_id].path
local_contents = rules.id_map[rule_id].contents
```

Reply:
Will need to be something similar to the following to address this. 
```python
            # Get local rule data if load_rule_loading is enabled. If not enabled rules variable will be None.
            local_rule: dict[str, Any] = params.get("rule", {})
            input_rule_id: str | None = None

            if local_rule:
                input_rule_id = cast("definitions.UUIDString", local_rule.get("rule_id"))

            if rules and input_rule_id and input_rule_id in rules.id_map:
                save_path = rules.id_map[input_rule_id].path
                local_contents = rules.id_map[input_rule_id].contents
                local_contents = rules.id_map[input_rule_id].contents
```